easybuild/easyconfigs/w/WildMagic/WildMagic-5.17-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = "MakeCp"
+
+name = 'WildMagic'
+version = '5.17'
+
+homepage = "https://www.geometrictools.com/index.html"
+description = """Wild Magic 5.17"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+source_urls = ['https://www.geometrictools.com/Downloads/']
+sources = ['WildMagic5p17.zip']
+checksums = ['b2caf7aa0c5309eb40965ed26439388dc1c93eb632312b40573f83e4746a9ecf']
+
+dependencies = [
+    ('binutils', '2.36.1'),
```

Comment:
@stefan-wolfsheimer Do you have a specific reason to list `binutils` as a runtime dependency here?

If not, this should be just a build dependency:

```python
builddependencies = [
    ('binutils', '2.36.1'),
]
dependencies = [
    ('freeglut', '3.2.1'),
]

Reply:
fixed in ceb214f